plugins/engine-configuration/assemblies/feature/src/main/feature/feature.xml

```diff
@@ -26,6 +26,7 @@
 
   <feature name="${project.artifactId}" version="${project.version}">
     <details>${project.description}</details>
+    <bundle>mvn:com.hitachivantara.security.web/csrf-token-service-client-java-httpclient-v4/${csrf.token.service.client.version}</bundle>
```

Comment:
Must use the property `hv-security-web.version` inherited from the parent pom, instead.

Reply:
addressed the comments